R/plot_recurrent_fusions.R

```diff
@@ -12,18 +12,31 @@
 #'
 #' @examples
 #' out_annofuse <- system.file("extdata", "PutativeDriverAnnoFuse_test_v14.tsv", package = "annoFuse")
-#' sfc <- read.delim(out_annofuse)
-#' plot_recurrent_fusions(sfc, groupby = "Fusion_Type", countID = "Sample")
+#' sfc <- read_tsv(out_annofuse)
+#' clinical<-read_tsv(system.file("extdata", "pbta-histologies.tsv", package = "annoFuse"))
+#' # Select only in-frame and frameshift
+#' sfc <- sfc %>%
+#' dplyr::filter(Fusion_Type %in% c("in-frame","frameshift"))
```

Comment:
Can we probably just add some indentations here to keep the example code straightforward readable?
I know in the roxygen portion it has to be added manually, but it is at least rendered correctly